### PR TITLE
Package nsq.0.4.0

### PR DESCRIPTION
**Pull request by ryanslade**
### `nsq.0.4.0`
A client library for the NSQ messaging platform

---
* Homepage: https://github.com/ryanslade/nsq-ocaml
* Source repo: git+https://github.com/ryanslade/nsq-ocaml.git
* Bug tracker: https://github.com/ryanslade/nsq-ocaml/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
Something is wrong:
```
#=== ERROR while compiling nsq.0.4.0 ==========================================#
# context              2.0.3 | linux/x86_64 | ocaml-base-compiler.4.07.1 | pinned(https://github.com/ryanslade/nsq-ocaml/archive/0.4.0.tar.gz)
# path                 ~/.opam/4.07/.opam-switch/build/nsq.0.4.0
# command              ~/.opam/4.07/bin/jbuilder build -p nsq -j 71
# exit-code            1
# env-file             ~/.opam/log/nsq-20-9f5627.env
# output-file          ~/.opam/log/nsq-20-9f5627.out
### output ###
# The jbuilder binary is deprecated and will cease to be maintained in July 2019.
# Please switch to dune instead.
# File "src/dune", line 4, characters 18-28:
# 4 |  (preprocess (pps ppx_expect ppx_sexp_conv ppx_compare ppx_deriving_yojson))
#                       ^^^^^^^^^^
# Error: Library "ppx_expect" not found.
# Hint: try: dune external-lib-deps --missing -j 71 -p nsq @install
```

File: packages/nsq/nsq.0.4.0/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "Ryan Slade <ryanslade@gmail.com>"
+authors: "Ryan Slade <ryanslade@gmail.com>"
+homepage: "https://github.com/ryanslade/nsq-ocaml"
+bug-reports: "https://github.com/ryanslade/nsq-ocaml/issues"
+dev-repo: "git+https://github.com/ryanslade/nsq-ocaml.git"
+build: [["jbuilder" "build" "-p" name "-j" jobs]]
```
Comment (kit-ty-kate) [inline packages/nsq/nsq.0.4.0/opam]: be aware that jbuilder syntax is currently being deprecated. See https://dune.build/blog/second-step-deprecation/

**Comment by ryanslade** [comment]
PTAL

**Comment by kit-ty-kate** [comment]
Thank you!

**Pull request merged**